src/user-guide/writing-workflows/scheduler.rst

```diff
@@ -0,0 +1,226 @@
+Scheduler Configuration
+=======================
+
+The :cylc:conf:`flow.cylc[scheduler]` section configures certain aspects of
+:term:`scheduler` behaviour at the workflow level.
+
+Many of these configurations can also be defined at the site-user level in
+the :cylc:conf:`global.cylc[scheduler]` section which is usually more
+appropriate.
```

Comment:
As a user this leaves me wondering "Why?". It might be better if the sentence ended after `global.cylc[scheduler]` and you then had a note saying - "It's often better to use a site or user config because..."